### PR TITLE
[BD-46] docs: added caret for ModalPopup component

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- Added caret for ModalPopup component.

### Deploy Preview

[ModalPopup component](https://deploy-preview-1361--paragon-openedx.netlify.app/components/modal/modal-popup/)

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [x] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [x] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [x] Were your changes tested in the `example` app?
* [x] Is there adequate test coverage for your changes?
* [x] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [x] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [x] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [x] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang! I've created [BLENDED-1307](https://openedx.atlassian.net/browse/BLENDED-1307) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

File: src/Modal/Modal.scss [hunk 1]
```diff
@@ -109,3 +109,43 @@
 .modal-footer {
   flex: 0 0 auto;
 }
+
+.pgn__modal-popup__tooltip {
+  margin: .5rem 0;
+  border: $border-width solid rgb(0 0 0 / 20%);
```
Comment (ihor-romaniuk) [inline src/Modal/Modal.scss:115]: Why you don't use `rgba(color, alpha)` instead of `rgb(color / alpha)`?

Comment (PKulkoRaccoonGang) [reply src/Modal/Modal.scss]: IDE says what to do)
Corrected, thanks

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1361&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2022-06-07+19:40&entry.1917517419=2022-06-08+10:49&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**